server/data/incidentClient.test.ts

```diff
@@ -80,12 +80,7 @@ test('getIncompleteReportsForReviewer', () => {
           where r.status = $1
           and   r.agency_id = $2
           order by r.incident_date`,
-    values: [
-      ReportStatus.SUBMITTED.value,
-      'agency-1',
-      StatementStatus.PENDING.value,
-      StatementStatus.REMOVAL_REQUESTED.value,
-    ],
+    values: [ReportStatus.SUBMITTED.value, 'agency-1', StatementStatus.PENDING.value],
```

Comment:
Just wondering, would the `StatementStatus.PENDING.value` be accessed here is there are only 2 attribute placeholders for each status and agency_id?